aws-logging-cloudwatch/src/main/java/com/amplifyframework/logging/cloudwatch/db/CloudWatchLoggingDatabase.kt

```diff
@@ -82,12 +83,15 @@ internal class CloudWatchLoggingDatabase(
 
     internal suspend fun bulkDelete(eventIds: List<Long>) = withContext(coroutineDispatcher) {
         contentUri
```

Comment:
Remove this line, should not have been here

Reply:
Got it, this part was from the original code before the change, left in its place because I did not understand why it's existing, and did not want to break it by removing what I do not understand... I've just tested out the operation is nominal without this line, removed!